Remora.Discord.Commands/Feedback/Services/FeedbackService.cs

```diff
@@ -39,11 +39,10 @@
 
 namespace Remora.Discord.Commands.Feedback.Services;
 
-/// <summary>
-/// Handles sending formatted messages to the users.
-/// </summary>
+/// <inheritdoc />
```

Comment:
Replace the documentation comments on the interface-implemented methods with inheritdoc as well.